.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -88,6 +88,134 @@ jobs:
+        run: sudo apt-get update && sudo apt-get install -y ninja-build clang-format-12
+      - name: Install Qt GCC
+        uses: jurplel/install-qt-action@v3
+        with:
+          aqtversion: '==3.1.*'
+          version: '6.2.4'
+          host: 'linux'
+          target: 'desktop'
+          arch: 'gcc_64'
+          cache: true
+      - name: Install Qt WASM
+        uses: jurplel/install-qt-action@v3
+        with:
+          aqtversion: '==3.1.*'
+          version: '6.2.4'
+          host: 'linux'
+          target: 'desktop'
+          arch: 'wasm_32'
+          tools: 'tools_cmake'
+          cache: true
+      - name: "Configure"
+        # FIXME: ninja is not being found?
+        env:
+          RUSTC_WRAPPER: sccache
+        run: |
+          source ./emsdk/emsdk_env.sh
+          /home/runner/work/cxx-qt/Qt/6.2.4/wasm_32/bin/qt-cmake -DQT_HOST_PATH=/home/runner/work/cxx-qt/Qt/6.2.4/gcc_64 -DBUILD_WASM=ON -Bbuild .
+      - name: "Build"
+        env:
+          RUSTC_WRAPPER: sccache
+        run: cmake --build build --parallel 4
+      - name: "Print compiler cache statistics"
+        run: sccache --show-stats
+      # This is a workaround for the fact that GH doesn't support updating existing caches.
+      # See: https://github.com/azu/github-actions-overwrite-cache-example
+      - name: "Delete previous compiler cache"
+        # Updating th cache doesn't work from forks
+        # So update it once it's merged into the repo
+        if: ${{ steps.compiler-cache-restore.outputs.cache-hit &&  github.event_name == 'push' }}
+        working-directory: /home/runner/cxx-qt
+        continue-on-error: true
+        run: |
+          gh extension install actions/gh-actions-cache
+          gh actions-cache delete "Ubuntu 22.04 (wasm_32) Qt6_compiler_cache" --confirm
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: "Save Compiler Cache"
+        # Updating th cache doesn't work from forks
+        # So update it once it's merged into the repo
+        if: ${{ github.event_name == 'push' }}
+        uses: actions/cache/save@v4
+        with:
+          path: /home/runner/.cache/sccache
+          key: "Ubuntu 22.04 (wasm_32) Qt6_compiler_cache"
+      - name: Test output files exist
+        run: |
+          test -f ./build/examples/qml_minimal/example_qml_minimal.html
+          test -f ./build/examples/qml_minimal/example_qml_minimal.js
+          test -f ./build/examples/qml_minimal/libqml_minimal.a
+          test -x ./build/examples/qml_minimal/example_qml_minimal.wasm
+          test -f ./build/examples/qml_minimal/qtloader.js
+          test -f ./build/examples/qml_features/example_qml_features.html
+          test -f ./build/examples/qml_features/example_qml_features.js
+          test -f ./build/examples/qml_features/libqml_features.a
+          test -x ./build/examples/qml_features/example_qml_features.wasm
+          test -f ./build/examples/qml_features/qtloader.js
```

Comment:
while making changes, move this test step to be after the `cmake --build` and before the sccache stats / tidying up of cache. So it's in the same order as other tests